great_expectations/rule_based_profiler/domain_builder/simple_column_suffix_domain_builder.py

```diff
@@ -68,8 +85,10 @@ def _get_domains(
             Domain(
                 domain_type=MetricDomainTypes.COLUMN,
                 domain_kwargs={
-                    "column": column_name,
-                    "batch_id": validator.active_batch_id,
+                    **{
+                        "column": column_name,
+                    },
```

Comment:
Why use this pattern instead of `"column": column_name`?

Reply:
@talagluck Good question -- this pattern improves code robustness.  We define the common components of the dictionary up front and then use to instantiate the final dictionary as a composition of input dictionaries (one of which contains the common component)  -- and this is the correct syntax (as supported by Python).  Thanks.

Reply:
I'd love to chat more about this. I can understand unpacking `metric_domain_kwargs` and passing it in, but creating a dictionary to immediately unpack it here feels confusing to me.